src/test/resources/org/prebid/server/it/openrtb2/nanointeractive/test-auction-nanointeractive-request.json

```diff
@@ -0,0 +1,118 @@
+{
+  "id": "tid",
+  "imp": [
+    {
+      "id": "impId13",
+      "banner": {
+        "format": [
+          {
+            "w": 300,
+            "h": 250
+          }
+        ],
+        "w": 300,
+        "h": 250
+      },
+      "ext": {
+        "nanointeractive": {
+          "pid": "58bfec94eb0a1916fa380163",
+          "nq": ["search query"],
+          "category": "Automotive",
+          "subId": "a23",
+          "ref": "https://nanointeractive.com"
+        }
+      }
+    },
+    {
+      "id": "impId001",
+      "banner": {
+        "format": [
+          {
+            "w": 300,
+            "h": 250
+          }
+        ],
+        "w": 300,
+        "h": 250
+      },
+      "ext": {
+        "nanointeractive": {
+          "pid": "58bfec94eb0a1916fa380163",
+          "nq": ["search query"],
+          "category": "Automotive",
+          "subId": "a23",
+          "ref": "https://nanointeractive.com"
+        }
+      }
+    }
+  ],
+  "device": {
+    "pxratio": 4.2,
+    "dnt": 2,
+    "language": "en",
+    "ifa": "ifaId"
+  },
+  "site": {
+    "page": "http://www.example.com",
+    "ref": "https://nanointeractive.com",
```

Comment:
remove this value if you want to check that smth was set. 
Also as I mentioned Site shouldn't be removed.